tgui/packages/tgui/interfaces/Cargo.js

```diff
@@ -370,57 +367,91 @@ const CargoCartButtons = (props, context) => {
         {cart.length >= 2 && cart.length + ' items'}{' '}
         {total > 0 && `(${formatMoney(total)} cr)`}
       </Box>
-      <Button
-        icon="times"
-        color="transparent"
-        content="Clear"
-        onClick={() => act('clear')}
-      />
+      {(!(requestonly || !can_send || !can_approve_requests) && (
+        <Button
+          icon="times"
+          color="transparent"
+          content="Clear"
+          onClick={() => act('clear')}
+        />
+      )) ||
+        ''}
```

Comment:
Shouldn't need to use this- is it displaying zeros?

Reply:
Nope I should have removed this. I'll do that